diff --git a/3.13-bone/am335x-bone.dts b/3.13-bone/am335x-bone.dts
index d8458a6..5e61ab7 100644
--- a/3.13-bone/am335x-bone.dts
+++ b/3.13-bone/am335x-bone.dts
@@ -597,22 +597,23 @@
 			ti,hwmods = "spi0";
 			dmas = <0xc 0x10 0xc 0x11 0xc 0x12 0xc 0x13>;
 			dma-names = "tx0", "rx0", "tx1", "rx1";
-			status = "okay";
-			pinctrl-names = "default";
-			pinctrl-0 = <0x10>;
-
-			spi@0 {
-				compatible = "spidev";
-				reg = <0x0>;
-				spi-max-frequency = <0xf42400>;
-				spi-cpha;
-			};
-
-			spi@1 {
-				compatible = "spidev";
-				reg = <0x1>;
-				spi-max-frequency = <0xf42400>;
-			};
+			status = "disabled";
+			//status = "okay";
+			//pinctrl-names = "default";
+			//pinctrl-0 = <0x10>;
+
+			//spi@0 {
+			//	compatible = "spidev";
+			//	reg = <0x0>;
+			//	spi-max-frequency = <0xf42400>;
+			//	spi-cpha;
+			//};
+
+			//spi@1 {
+			//	compatible = "spidev";
+			//	reg = <0x1>;
+			//	spi-max-frequency = <0xf42400>;
+			//};
 		};
 
 		spi@481a0000 {
diff --git a/3.13-bone/am335x-boneblack.dts b/3.13-bone/am335x-boneblack.dts
index 2f9a3d1..7da1f8c 100644
--- a/3.13-bone/am335x-boneblack.dts
+++ b/3.13-bone/am335x-boneblack.dts
@@ -616,22 +616,23 @@
 			ti,hwmods = "spi0";
 			dmas = <0xc 0x10 0xc 0x11 0xc 0x12 0xc 0x13>;
 			dma-names = "tx0", "rx0", "tx1", "rx1";
-			status = "okay";
-			pinctrl-names = "default";
-			pinctrl-0 = <0x11>;
-
-			spi@0 {
-				compatible = "spidev";
-				reg = <0x0>;
-				spi-max-frequency = <0xf42400>;
-				spi-cpha;
-			};
-
-			spi@1 {
-				compatible = "spidev";
-				reg = <0x1>;
-				spi-max-frequency = <0xf42400>;
-			};
+			status = "disabled";
+			//status = "okay";
+			//pinctrl-names = "default";
+			//pinctrl-0 = <0x11>;
+
+			//spi@0 {
+			//	compatible = "spidev";
+			//	reg = <0x0>;
+			//	spi-max-frequency = <0xf42400>;
+			//	spi-cpha;
+			//};
+
+			//spi@1 {
+			//	compatible = "spidev";
+			//	reg = <0x1>;
+			//	spi-max-frequency = <0xf42400>;
+			//};
 		};
 
 		spi@481a0000 {
