diff --git a/3.13-bone/am335x-bone.dts b/3.13-bone/am335x-bone.dts
index baeb212..5285720 100644
--- a/3.13-bone/am335x-bone.dts
+++ b/3.13-bone/am335x-bone.dts
@@ -283,10 +283,10 @@
 			clock-frequency = <0x2dc6c00>;
 			reg = <0x48024000 0x2000>;
 			interrupts = <0x4a>;
-			status = "disabled";
-			//status = "okay";
-			//pinctrl-names = "default";
-			//pinctrl-0 = <0x6>;
+			//status = "disabled";
+			status = "okay";
+			pinctrl-names = "default";
+			pinctrl-0 = <0x6>;
 		};
 
 		serial@481a6000 {
diff --git a/3.13-bone/am335x-boneblack.dts b/3.13-bone/am335x-boneblack.dts
index 70c6f16..a4e79d9 100644
--- a/3.13-bone/am335x-boneblack.dts
+++ b/3.13-bone/am335x-boneblack.dts
@@ -297,10 +297,10 @@
 			clock-frequency = <0x2dc6c00>;
 			reg = <0x48024000 0x2000>;
 			interrupts = <0x4a>;
-			status = "disabled";
-			//status = "okay";
-			//pinctrl-names = "default";
-			//pinctrl-0 = <0x6>;
+			//status = "disabled";
+			status = "okay";
+			pinctrl-names = "default";
+			pinctrl-0 = <0x6>;
 		};
 
 		serial@481a6000 {
