diff --git a/3.13-bone/am335x-bone.dts b/3.13-bone/am335x-bone.dts
index 0c43800..f9cf2ff 100644
--- a/3.13-bone/am335x-bone.dts
+++ b/3.13-bone/am335x-bone.dts
@@ -415,11 +415,11 @@
 			ti,hwmods = "i2c2";
 			reg = <0x4802a000 0x1000>;
 			interrupts = <0x47>;
-			status = "disabled";
-			//status = "okay";
-			//pinctrl-names = "default";
-			//pinctrl-0 = <0xa>;
-			//clock-frequency = <0x186a0>;
+			//status = "disabled";
+			status = "okay";
+			pinctrl-names = "default";
+			pinctrl-0 = <0xa>;
+			clock-frequency = <0x61a80>;
 		};
 
 		i2c@4819c000 {
diff --git a/3.13-bone/am335x-boneblack.dts b/3.13-bone/am335x-boneblack.dts
index 582ba1d..6e93d7c 100644
--- a/3.13-bone/am335x-boneblack.dts
+++ b/3.13-bone/am335x-boneblack.dts
@@ -429,11 +429,11 @@
 			ti,hwmods = "i2c2";
 			reg = <0x4802a000 0x1000>;
 			interrupts = <0x47>;
-			status = "disabled";
-			//status = "okay";
-			//pinctrl-names = "default";
-			//pinctrl-0 = <0xa>;
-			//clock-frequency = <0x186a0>;
+			//status = "disabled";
+			status = "okay";
+			pinctrl-names = "default";
+			pinctrl-0 = <0xa>;
+			clock-frequency = <0x61a80>;
 		};
 
 		i2c@4819c000 {
