diff --git a/3.13-bone/am335x-bone.dts b/3.13-bone/am335x-bone.dts
index 8dc0222..9a2bc75 100644
--- a/3.13-bone/am335x-bone.dts
+++ b/3.13-bone/am335x-bone.dts
@@ -433,11 +433,11 @@
 			ti,hwmods = "i2c2";
 			reg = <0x4802a000 0x1000>;
 			interrupts = <0x47>;
-			status = "disabled";
-			//status = "okay";
-			//pinctrl-names = "default";
-			//pinctrl-0 = <0xa>;
-			//clock-frequency = <0x186a0>;
+			//status = "disabled";
+			status = "okay";
+			pinctrl-names = "default";
+			pinctrl-0 = <0xa>;
+			clock-frequency = <0x186a0>;
 		};
 
 		i2c@4819c000 {
diff --git a/3.13-bone/am335x-boneblack.dts b/3.13-bone/am335x-boneblack.dts
index 719bb6e..1172413 100644
--- a/3.13-bone/am335x-boneblack.dts
+++ b/3.13-bone/am335x-boneblack.dts
@@ -447,11 +447,11 @@
 			ti,hwmods = "i2c2";
 			reg = <0x4802a000 0x1000>;
 			interrupts = <0x47>;
-			status = "disabled";
-			//status = "okay";
-			//pinctrl-names = "default";
-			//pinctrl-0 = <0xa>;
-			//clock-frequency = <0x186a0>;
+			//status = "disabled";
+			status = "okay";
+			pinctrl-names = "default";
+			pinctrl-0 = <0xa>;
+			clock-frequency = <0x186a0>;
 		};
 
 		i2c@4819c000 {
