diff --git a/3.13-bone/am335x-bone.dts b/3.13-bone/am335x-bone.dts
index baeb212..97e372d 100644
--- a/3.13-bone/am335x-bone.dts
+++ b/3.13-bone/am335x-bone.dts
@@ -316,10 +316,10 @@
 			clock-frequency = <0x2dc6c00>;
 			reg = <0x481aa000 0x2000>;
 			interrupts = <0x2e>;
-			status = "disabled";
-			//status = "okay";
-			//pinctrl-names = "default";
-			//pinctrl-0 = <0x8>;
+			//status = "disabled";
+			status = "okay";
+			pinctrl-names = "default";
+			pinctrl-0 = <0x8>;
 		};
 
 		i2c@44e0b000 {
